### ISSUE TITLE
Master Server deployment

### ISSUE THREAD

**Issue by jonlab** [jonlab/NewAtlantis#10]
Do we need this ?

**Comment by rolandcahen** [comment]
Could someone do this, but not Jonathan. This should be checked with the new network system

**Comment by jonlab** [comment]
If we go on the new Unity 5.1 networking system, we should use the "Internet Services" instead : 
http://docs.unity3d.com/Manual/UNetInternetServicesOverview.html

**Comment by rolandcahen** [comment]
Does it have any chance to solve our ENSCI connexion problem ?

**Comment by jonlab** [comment]
No, I don't think so, but maybe we could try a working example that we could test at the ENSCI.
